src/libslic3r/Fill/Fill.cpp

```diff
@@ -1462,6 +1462,7 @@ void Layer::make_ironing()
 		double 		height;
 		double 		speed;
 		double 		angle;
+        bool        fixed_angle;
```

Comment:
I think `is_using_template_angle` does the same job. I think we could just keep one of them. I personally prefer the naming of `fixed_angle`. Maybe we can rename `is_using_template_angle` to `fixed_angle` instead.